.github/workflows/es-yearly-globals.yml

```diff
@@ -0,0 +1,24 @@
+name: Create ES yearly globals
+
+on:
+  workflow_dispatch:
+  schedule:
+    # “At 00:00 on day-of-month 1 in July.” https://crontab.guru/#0_0_1_7_*
+    - cron: "0 0 1 7 *"
+
+jobs:
+  update:
+    if: github.event_name != 'schedule' || github.repository == 'sindresorhus/globals'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+      - run: npm install
+      - run: node scripts/create-yearly-es-globals.mjs
+      - run: npm run build
+      - uses: peter-evans/create-pull-request@v6
+        with:
+          commit-message: Add `es2026` globals
```

Comment:
Doesn't this mean we have to update the workflow each year, which kinda defeats the purpose?